frontend/src/app/Settings/Facility/facilitySchema.ts

```diff
@@ -27,6 +30,18 @@ function orderingProviderIsRequired(
   return true;
 }
 
+function isValidNpi(
+  this: yup.TestContext<Record<string, any>>,
+  input = ""
+): boolean {
+  if (this?.options?.context?.orderingProviderIsRequired) {
+    // console.log("NPI is required so it better be good");
```

Comment:
You might want to clear out this unneeded `console.log` call, since it's commented out.

Reply:
Whoops, thank you! Snuck in there during testing